assets/css/cinegraph_neutral.css

```
@@ -0,0 +1,8 @@
+/* Cinegraph Neutral theme — standalone Tailwind build.
+ * Intentionally no app.css imports — keeps the design system fully isolated
+ * from the main app stylesheet so existing pages render unchanged.
+ */
+
+@tailwind base;
```

Stylelint (17.9.0)

[error] Unexpected unknown at-rule "@tailwind" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)